### DISCUSSION TITLE
Welcome to PaddleSlim Discussions!

### DISCUSSION THREAD

**Discussion by yghstill** [PaddlePaddle/PaddleSlim#1389] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by PlumBlossomMaid** [comment]
跟用过的兄弟姐妹们避个坑，如果用这个paddleslim库的话还想要导出为onnx，千万不要用量化的模型导出，不要等到辛辛苦苦训练完模型之后才发现无法导出为ONNX。裁剪的话paddleslim只是对模型的某些权重进行了裁剪并没有动模型的算子，我这边还是可以导出onnx的。
之所以量化不支持导出onnx，我的猜测是量化之后插入的算子根本无法与onnx算子对齐而不是paddle2onnx的作者懒得弄，因为我发现使用同类框架Pytorch进行量化的模型也是无法被导出onnx的，也是显示算子不支持。如果想要实现模型被导出onnx且被优化，可以试试使用onnxslim这个库进行模型压缩哦😉。
